src/ResidentDirectory.js

```diff
@@ -26,7 +26,7 @@ const colorStylesResident = {
             backgroundColor: isFocused ? '#B2D4FF' : 'white',
             color: (data.inResidence != null && !data.inResidence) ? 'red' : 'black',
             ":active": {
-                backgroundColor: "#B2D4FF"
+                backgroundColor: "white"//"#B2D4FF"
```

Comment:
Does this not mess with the highlighting?

Reply:
I was playing around and forgot to revert -- will switch it back to #B2D4FF